tests/security/cc/remove_bridge_network.pm

```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Remove bridge network in CC system role for multi-machine test
+# Maintainer: Liu Xiaojing <xiaojing.liu@suse.com>
+# Tags: poo#102116
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = shift;
+
+    select_console 'root-console';
+
+    assert_script_run('rm -f /etc/sysconfig/network/ifcfg-br0');
+    assert_script_run('sed -i \'s/none/dhcp/\' /etc/sysconfig/network/ifcfg-eth0');
```

Comment:
Xiaojing, 

It is not good way to use hardcode "eth0". please use "bridge" cmd to get the right NIC name and bridge name.